contest/views.py

```diff
@@ -113,11 +115,11 @@ def contest(request, cid):
 
 @login_required
 def new(request):
+    title = "New Contest"
     if can_create_contest(request.user):
         if request.method == 'GET':
             form = ContestForm(initial=\
-                {'user':request.user, 'method':request.method})
-            title = "New Contest"
+                {'owner':request.user, 'user':request.user, 'method':request.method})
```

Comment:
Why do we need 'user' in a new contest form?
I mean.. the user must be owner, is it necessary to pass same value twice?